### PR TITLE
Crispina Muriel

### PR DESCRIPTION
**Pull request by ghost**
yay not late

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,33 +2,76 @@
...
-    
-2. 
+
+    class Human {
+     constructor(name, age){
+       this.name = name;
+       this.age = age;
+     }
+
+     ageOneYear(str){
```
Comment (michellelala) [inline README.md:19]: No argument is needed for this function! It doesn't hurt the function, just for future reference :)

File: README.md [hunk 2]
```diff
@@ -2,33 +2,76 @@
...
+       return this.age++
+     }
+
+      eating(str){
```
Comment (michellelala) [inline README.md]: No argument needed

File: README.md [hunk 3]
```diff
@@ -2,33 +2,76 @@
...
+        return "mmm, mmm, mmm, I'm love'n it."
+      }
+    }
+
```
Comment (michellelala) [inline README.md:27]: "Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it"."
Just a couple of notes for your eating function:
- no argument is needed
- your function returns the string, but we want to log it with console.log instead
- see comment for line 33!

File: README.md [hunk 4]
```diff
@@ -2,33 +2,76 @@
...
+
+    let daniel = new Human("Daniel", 34)
+    console.log(daniel.age)
+    daniel.ageOneYear()
+    console.log(daniel.age)
+    console.log(daniel.eating())
```
Comment (michellelala) [inline README.md:33]: This way, you only need to call on the function with `daniel.eating()` to log it into the console. Please see me if you have any questions about this or if I'm not explaining it clearly! :) 

Small note: variable names that are classes should begin with an uppercase.
Great job with this assignment!

File: README.md [hunk 5]
```diff
@@ -2,33 +2,76 @@
...
-// => Vector {x: 3, y: 5}
 console.log(v1.minus(v2));
-// => Vector {x: -1, y: -1}
-
-var v3 = new Vector(3, 4)
-console.log(v3.getLength());
-// => 5
-```
+console.log(v2.getLength());
+//holy god, you guys, it worked
```
Comment (michellelala) [inline README.md:77]: Awesome job!

Comment (XavierC4Q) [inline README.md:77]: Nitpick, but your variable names for your plus and minus functions should be more clear. Something like sumX, or sumY where it is clear what the values represent.

**Pull request closed without merging**